app/utils/paginator.js

```diff
@@ -168,6 +169,30 @@ Paginator.prototype = {
     return namedRange
   },
 
+  buildNavigationLinks: function (_links, resultsLength) {
```

Comment:
Very minor but this method could be named
```suggestion
  buildNavigationLinks: function buildNavigationLinks(_links, resultsLength) {
```

Also thinking about it there's no reason this needs to use an opaque name like `_links`, it could be more specific, I'll include a potential option but I don't think it's necessarily the right one:
```suggestion
  buildNavigationLinks: function buildNavigationLinks(resultsGeneratedPageLinks, resultsLength) {
```

Reply:
👍 think I was in the zone of json. Updated!